clients/src/main/java/org/apache/kafka/clients/MetadataUpdater.java

```diff
@@ -82,4 +82,9 @@
      * start of the update if possible (see `maybeUpdate` for more information).
      */
     void requestUpdate();
+
+    /**
+     * Close this updater.
+     */
+    void close();
```

Comment:
Not a big deal, but perhaps we could let `MetadataUpdater` implement `Closeable`? We can still override `close()` so that it doesn't throw an exception.